src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useState } from "react";
+import { ConfigContext } from "Context";
+import { useAsync } from "react-use";
+import Loader from "components/Loader";
+import { FormattedMessage, useIntl } from "react-intl";
+import { v4 as uuidv4 } from "uuid";
+import {
+  ButtonGroup,
+  Heading,
+  LinkButton,
+} from "@utrecht/component-library-react";
+import { get, put } from "../../api";
+import PropTypes from "prop-types";
```

Comment:
unfortunately we don't have an automated tool for this (yet), but please group your imports similar to the backend:

1. Third party packages (from `node_modules`)
2. blank line
3. Own package (absolute imports from the `src` folder)
4. blank line
5. Relative imports within the same (sub) package (e.g. `./utils`)